misk/src/main/kotlin/misk/digester/PrometheusHistogram.kt

```diff
@@ -0,0 +1,13 @@
+package misk.digester
+
+class PrometheusHistogram: Histogram {
+    lateinit var histogram: io.prometheus.client.Histogram
+
+    override fun record(vararg  labelValues: String): io.prometheus.client.Histogram.Child {
```

Comment:
oh we should remove this method. it's meant to be a Builder pattern which we no longer support with the new interface